tests/unit/test_bert_sentence_encoding.py

```diff
@@ -19,14 +19,15 @@ def data():
 
 
 @pytest.mark.cpu
```

Comment:
we don't have a cpu tag, it is the absence of the other tags, please see this line: https://github.com/microsoft/nlp-recipes/blob/master/tests/ci/cpu_unit_tests_linux.yml#L46 We can erase `@pytest.mark.cpu`